rwengine/src/objects/VehicleObject.cpp

```diff
@@ -227,9 +227,15 @@ void VehicleObject::setupModel() {
 
     auto compRules = vehicleInfo->componentrules_;
     auto numComponents = [](int rule) {
-        if ((rule & 0xFFF) == 0xFFF) return 0;
-        if ((rule & 0xFF0) == 0xFF0) return 1;
-        if ((rule & 0xF00) == 0xF00) return 2;
+        if ((rule & 0xFFF) == 0xFFF) {
```

Comment:
nit: maybe name this values with enum?

Reply:
Do you think about variable? I was thinking just about values `0xFFF`, `0xFF0`, `0xF00`. Maybe they have some meaning?

Reply:
I have no idea what their meaning is. So I would just leave it the same.